### DISCUSSION TITLE
National Geographic Wild HD East

### DISCUSSION THREAD

**Discussion by jphilippe2017** [iptv-org/community#743] [Q&A]
**Channel Name:** Nat Geo Wild
**Channel ID:** NationalGeographicWildHDEast.us
**Country:** USA
**Language:** English
**Website:** http://www.natgeotv.com
**Stream URL (optional):** 
**Notes (optional):** HD please!

**Comment by F640** [comment]
https://livecdn.fptplay.net/hda3/natgeowild_hls.smil/chunklist.m3u8
Asia or East version?
Should work without referrer or referrer set to "https://fpyplay.vn/"

**Comment by AntiPontifex** [comment]
Dead link: please close.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19731
